tests/roles/nova_adoption/tasks/nova_ironic.yaml

```diff
@@ -0,0 +1,130 @@
+                  [workarounds]
+                  disable_compute_service_check_for_ffu=true
+            cell1:
+              conductorServiceTemplate:
+                customServiceConfig: |
+                  [workarounds]
+                  disable_compute_service_check_for_ffu=true
+              novaComputeTemplates:
+                standalone:
+                  customServiceConfig: |
+                    [DEFAULT]
+                    host = standalone.localdomain
+                    [workarounds]
+                    disable_compute_service_check_for_ffu=true
+                  replicas: 1
+                  resources: {}
+                  computeDriver: ironic.IronicDriver
+                  networkAttachments:
+                    - internalapi
+    '
+
+- name: wait for Nova control plane services' CRs to become ready
+  ansible.builtin.include_tasks:
+    file: wait.yaml
+
+- name: Remove FFU workarounds
+  ansible.builtin.shell: |
+    {{ shell_header }}
+    {{ oc_header }}
+    oc patch openstackcontrolplane openstack -n openstack --type=merge --patch '
+    spec:
+      nova:
+        template:
+          apiServiceTemplate:
+            customServiceConfig: |
+              [workarounds]
+              disable_compute_service_check_for_ffu=false
+          metadataServiceTemplate:
+            customServiceConfig: |
+              [workarounds]
+              disable_compute_service_check_for_ffu=false
+          schedulerServiceTemplate:
+            customServiceConfig: |
+              [workarounds]
+              disable_compute_service_check_for_ffu=false
+          cellTemplates:
+            cell0:
+              conductorServiceTemplate:
+                customServiceConfig: |
+                  [workarounds]
+                  disable_compute_service_check_for_ffu=false
+            cell1:
+              conductorServiceTemplate:
+                customServiceConfig: |
+                  [workarounds]
+                  disable_compute_service_check_for_ffu=false
+              novaComputeTemplates:
+                standalone:
+                  customServiceConfig: |
+                    [DEFAULT]
+                    host = standalone.localdomain
+                    [workarounds]
+                    disable_compute_service_check_for_ffu=false
+    '
+
+- name: wait for Nova control plane services' CRs to become ready
+  ansible.builtin.include_tasks:
+    file: wait.yaml
+
+# TODO(bogdando): implement podified FQDNs check for Nova endpoints
+- name: check that Nova endpoints are defined and pointing to the podified FQDNs and that Nova API responds
+  ansible.builtin.shell: |
+    {{ shell_header }}
+    {{ oc_header }}
+    alias openstack="oc exec -t openstackclient -- openstack"
+
+    ${BASH_ALIASES[openstack]} endpoint list | grep nova
+    ${BASH_ALIASES[openstack]} server list
+  register: nova_responding_result
+  until: nova_responding_result is success
+  retries: 60
+  delay: 2
```

Comment:
I would  also verify that:
* after adoption the an existing baremetal instance can be still managed? 
* after adoption an adopted baremetal node can be used to deploy a baremetal instance?

I'm OK to add these test (if make sense) in follow up PR

Reply:
do we need to duplicate this generic for Nova API checks? We already have them for nova adoption, perhaps here should be ones specific to Ironic for Nova. Please also update TODO

Reply:
I moved the check for Nova API's to a separate file and update both `nova_ironic.yaml` and `nova_libvirt.yaml` tasks file to use that.

I also added a test deploying another instance on adopted baremetal. I have not got that to pass yet because data in swift is not migrated.